backend/server.py

```diff
@@ -23,5 +25,28 @@ def get_data_by_id(census_id):
     data = db.get_census_division_data(census_id)
     return jsonify({ "error": "", "data": data }) 
 
+@app.route("/users/new")
+def create_user():
+    # Check if proper params have been passed in
+    params = ['email', 'password', 'admin']
+    for param in params:
+        if not request.args.get(param, None):
+            return jsonify({"error": "Missing param: {}".format(param)})
+
+    email = str(request.args.get('email', ''))
+    password = str(request.args.get('password', ''))
+    admin = bool(request.args.get('admin'))
+
+    try:
+        user = auth.create_user(email=email, password=password)
+        auth.set_custom_user_claims(user.uid, {'admin': admin})
+    except auth.AuthError:
+        return jsonify({"error": "User with email {} already exists".format(email)})
+
+    return jsonify({'uid': user.uid})
+
 if __name__ == "__main__":
+    cred = credentials.Certificate("instance/c4k-dashboard-firebase-adminsdk-ypbc3-c66b8c5a1c.json")
```

Comment:
this is probably not good... should look into alternatives for when we put it into production